### ISSUE TITLE
Retest Only: Right Menu Pane- EPA 508 issue 

### ISSUE THREAD

**Issue by JanellC** [US-EPA-CAMD/PI-1#2284]
## Finding 

| Page/ Screen | Finding | WCAG 2.0 Standard(s) |
|---|---|---|
|  Right Menu Pane | In the right menu pane, when one of the submenus is expanded, after navigating to the last submenu item, focus then moves to the next top menu item.   (For example, when Environmental Topics is expanded, the tab order goes from A-Z topic index to Laws and Regulations then About EPA even though these items are hidden.) | 2.4.3 |

## Context 
The tech team is working on the right menu component. The new menu component will be implemented with ticket Refactor CAMPD UI to use easey-design-system components #1572. This ticket should be retested with ticket #1572

**Comment by aprematta** [comment]
retest complete, there are no more expandable elements. Issue has been fixed
